contact-discovery-app/README.md

```diff
@@ -0,0 +1,54 @@
+# Contact discovery application
+
+This is a sample application of a contact discovery service.
```

Comment:
A link to the Signal blog post that this sample is drawn from would be good, along with a brief statement of differences.